lib/input/input.go

```diff
@@ -20,6 +21,12 @@ func GetInput() {
 	for {
 		line, err = reader.ReadString('\n')
 		if err != nil {
+			// if stdin is unavailable (msh running as service)
+			// exit from input goroutine to avoid an infinite loop
+			if err == io.EOF {
+				errco.LogMshErr(errco.NewErr(errco.INPUT_UNAVAILABLE_ERROR, errco.LVL_D, "GetInput", "stdin unavailable, exiting input goroutine"))
```

Comment:
I'd put it on LVL_B as I think it'd be important to know even if not in debug mode (equivalent to info/warn in normal log levels).

Reply:
I don't think this is the case, on LVL_B there are only logs for minecraft server status, config loading and os support.

Even tho this bugfix is extremely important, the input goroutine exiting should only be reported as LVL_D (deep logging) (LVL_C is reserved for minecraft server logs)

if users dont know why the input functionality disappears, they can change the log level